src/verilog/verilog_expr.h

```diff
@@ -322,6 +327,13 @@ class verilog_parameter_declt : public verilog_module_itemt
     {
       return static_cast<exprt &>(add(ID_value));
     }
+
+    // helper to generate a symbol expression
+    symbol_exprt symbol_expr() const
+    {
+      return symbol_exprt(identifier(), type())
```

Comment:
Might use `return symbol_exprt::typeless(identifier())`

Reply:
Ah, note it's `type()`, not `typet()`. It's the type of the declarator, not the empty type.